reporting-guide.md

```diff
@@ -1,18 +1,37 @@
 
-## Reports Endpoint User Guide
+---
```

Comment:
Is this header going to show up in the docs or did you find a way to give commands to adobe.io?

Reply:
There are specific instructions on the tech writers wiki for Adobe I/O I am following. I assume they will work. If they don't we can fix them > 
"Please note that a few items contain markdown specific for Adobe I/O rendering (not Github), such as the SEO info. This is per the "collaborative documentation model" instructions."

Reply:
Can you send me a link to those instructions?